contracts/strategies/algebra/AlgebraConverterStrategyLogicLib.sol

```diff
@@ -488,10 +495,15 @@ library AlgebraConverterStrategyLogicLib {
   function needStrategyRebalance(PairBasedStrategyLogicLib.PairState storage pairState, ITetuConverter converter_) external view returns (
     bool needRebalance
   ) {
+    address pool = pairState.pool;
+    uint poolPriceDecimals = IERC20Metadata(pairState.tokenA).decimals();
+    uint poolPriceAdjustment = poolPriceDecimals < 18 ? 10 ** (18 - poolPriceDecimals) : 1;
```

Comment:
it worth to move poolPriceAdjuctment calculation to separate helper pure function because it repeats in several places in each pair-strategy

Reply:
also add comment that we assume that decimals never higher than 18